distage-example-monofunctor-tf/src/main/resources/META-INF/native-image/auto-cold/reflect-config.json

```diff
@@ -0,0 +1,400 @@
+[
```

Comment:
Please put Graal Native configs in a separate module and depend on that module, instead of copying.

Reply:
Totally forgot about such option 🔝